### ISSUE TITLE
Feature: download hazard curves

### ISSUE THREAD

**Issue by chrisdicaprio** [GNS-Science/simple-toshi-ui#274]
As a user I want to be able to download the hazard curves as a CSV file. The download will include data for all PGA/SA Periods available. The location, forecast time-span, background seismicity, GMM should be as selected for the plot.

**Comment by chrisdicaprio** [comment]
the output from the API query is:
```
{
  "data": {
    "node": {
      "name": "Inversion Solution Site Hazard",
      "created": "2021-12-18T23:44:26.745888+00:00",
      "column_types": [
        "double",
        "string",
        "double",
        "string",
        "string",
        "double",
        "double",
        "double",
        "double"
      ],
      "column_headers": [
        "forecast_timespan",
        "bg_seismicity",
        "iml_period",
        "gmpe",
        "location",
        "lat",
        "lon",
        "x",
        "y"
      ],
      "rows": [
        [
          "50",
          "EXCLUDE",
          "0.0",
          "ASK_2014",
          "Wellington",
          "-41.276825",
          "174.777969",
          "0.00010000000000000009",
          "0.768192160998513"
        ],
        [
          "50",
          "EXCLUDE",
          "0.0",
          "ASK_2014",
          "Wellington",
          "-41.276825",
          "174.777969",
          "0.0001299999999999999",
          "0.7681921609985128"
        ],
        [
          "50",
          "EXCLUDE",
          "0.0",
          "ASK_2014",
          "Wellington",
          "-41.276825",
          "174.777969",
          "0.00016",
          "0.768192160998512"
        ],
        [

etc
```
The user will have already chosen a `"forecast_timespan","bg_seismicity","gmpe","location"`
The `location` will match a `"lat", "lon"`
Therefore, none of those will change for a single download.

The table structure should be
| Acceleration | PoE (PGA) | PoE (0.1s) | PoE (0.2s) | etc for all `iml_period` |
| --- | --- | --- | --- | --- |
| x | y | y | y | y |

Each PoE column corresponds to an `iml_period`
PoE (PGA) --> `iml_period` = 0
PoE (0.1s) --> `iml_period` = 0.1
PoE (0.2s) --> `iml_period` = 0.1
etc
Each row is unique `x` entry in the query result.

In other words each curve in the figure below is a column of the downloaded file
![Untitled presentation(4)](https://user-images.githubusercontent.com/91439467/159192682-f4e7d726-26ec-4e29-aa7e-8b19295b735e.jpg)
:
